ethcore/src/executive.rs

```diff
@@ -245,30 +246,51 @@ impl<'a> Executive<'a> {
 		}
 		trace!("Executive::call(params={:?}) self.env_info={:?}", params, self.info);
 
+		let delegate_call = params.code_address != params.address;
```

Comment:
delegate calls are not traced further - if A delegatecalls B delegatecalls A, then, correctly, nothing will be traced.